Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
     "shuttle",
+    "shuttle_rand_08",
```

Comment:
Why are we putting `_08` in the name?

Reply:
We're not, we're putting `_08` in the path. In the future we'll have an `_08` and an `_09`, both of which have `shuttle-rand` in their `Cargo.toml`, but one publishes a mirror of rand 0.8, and one publishes a mirror of rand 0.9.

Reply:
can we make it `_0_8`?

Reply:
Why?

Reply:
Oh getcha